hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/repair/ldb/RocksDBManualCompaction.java

```diff
@@ -0,0 +1,85 @@
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hadoop.ozone.repair.ldb;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.hadoop.hdds.cli.HddsVersionProvider;
+import org.apache.hadoop.hdds.utils.IOUtils;
+import org.apache.hadoop.hdds.utils.db.managed.ManagedCompactRangeOptions;
+import org.apache.hadoop.hdds.utils.db.managed.ManagedRocksDB;
+import org.apache.hadoop.ozone.debug.RocksDBUtils;
+import org.apache.hadoop.ozone.repair.RepairTool;
+import org.rocksdb.ColumnFamilyDescriptor;
+import org.rocksdb.ColumnFamilyHandle;
+import org.rocksdb.RocksDBException;
+import picocli.CommandLine;
+
+/**
+ * Tool to perform compaction on a table.
+ */
+@CommandLine.Command(
+    name = "compact",
+    description = "CLI to compact a column-family in the DB. " +
+        "Note: If om.db is compacted then it will impact efficient snapshot diff.",
+    mixinStandardHelpOptions = true,
+    versionProvider = HddsVersionProvider.class
+)
+public class RocksDBManualCompaction extends RepairTool {
+
+  @CommandLine.Option(names = {"--db"},
+      required = true,
+      description = "Database File Path")
+  private String dbPath;
+
+  @CommandLine.Option(names = {"--column_family", "--column-family", "--cf"},
+      required = true,
+      description = "Column family name")
+  private String columnFamilyName;
+
+  @Override
+  public void execute() throws Exception {
+    List<ColumnFamilyHandle> cfHandleList = new ArrayList<>();
+    List<ColumnFamilyDescriptor> cfDescList = RocksDBUtils.getColumnFamilyDescriptors(
+        dbPath);
+
+    try (ManagedRocksDB db = ManagedRocksDB.open(dbPath, cfDescList, cfHandleList)) {
+      ColumnFamilyHandle cfh = RocksDBUtils.getColumnFamilyHandle(columnFamilyName, cfHandleList);
+      if (cfh == null) {
+        throw new IllegalArgumentException(columnFamilyName +
+            " is not in a column family in DB for the given path.");
+      }
+
+      info("Running compaction on " + columnFamilyName);
+      if (!isDryRun()) {
+        ManagedCompactRangeOptions compactOptions = new ManagedCompactRangeOptions();
+        compactOptions.setBottommostLevelCompaction(ManagedCompactRangeOptions.BottommostLevelCompaction.kForce);
+        db.get().compactRange(cfh, null, null, compactOptions);
+      }
+      info("Compaction completed.");
+
+    } catch (RocksDBException exception) {
+      error("Failed to compact the RocksDB for the given path: %s, column-family:%s", dbPath, columnFamilyName);
+      error("Exception: " + exception);
```

Comment:
Should we just throw an exception with this message instead? The second "Failed to compact RocksDB" message seems redundant, but throwing something is necessary to make the command exit non-zero. Current output is rendered correctly without the stack trace though:
```
$ ./ozone repair ldb compact --db=lkjhlk --column-family=foo
ATTENTION: Running as user foo. Make sure this is the same user used to run the Ozone process. Are you sure you want to continue (y/N)? y
Run as user: foo
Failed to compact the RocksDB for the given path: lkjhlk, column-family:foo
Exception: org.rocksdb.RocksDBException: lkjhlk/CURRENT: does not exist (create_if_missing is false)
Failed to compact RocksDB.
```